tools/reviewer/local_server/service/tests/CodeReviewServiceGetDiffFilesTest.java

```diff
@@ -0,0 +1,435 @@
+  private FirestoreClient firestoreClient = mock(FirestoreClient.class);
+
+  @Before
+  public void setup() throws IOException {
+    Flags.parse(
+        new String[0], AuthService.class.getPackage(), CodeReviewService.class.getPackage());
+    component =
+        DaggerCodeReviewServiceGetDiffFilesTest_TestComponent.builder()
+            .aaModule(
+                new AaModule() {
+                  @Provides
+                  @Singleton
+                  @Override
+                  @Named("Base path")
+                  public String provideBasePath(FileUtils fileUtils) {
+                    return aaBaseFolder;
+                  }
+                })
+            .build();
+    gitRepoFactory = component.getGitRepoFactory();
+    fileUtils = component.getFileUtils();
+    String testFolder = Files.createTempDirectory("temp").toAbsolutePath().toString();
+    String initialRepoFolder = fileUtils.joinToAbsolutePath(testFolder, "initial_repo");
+    aaBaseFolder = fileUtils.joinToAbsolutePath(testFolder, "base_folder");
+
+    when(firestoreClientFactory.create(any(), any())).thenReturn(firestoreClient);
+
+    codeReviewService =
+        new CodeReviewService(
+            component.getAuthService(),
+            fileUtils,
+            aaBaseFolder,
+            gitRepoFactory,
+            component.getTextDifferencer(),
+            firestoreClientFactory);
+
+    createInitialRepo(initialRepoFolder);
+    initAaBase(initialRepoFolder, aaBaseFolder);
+    createAaWorkspace(TEST_WORKSPACE);
+    createBlockingStub();
+    writeFile(TEST_FILE_CONTENTS);
+    testFileCommitId = repo.commit(repo.getUncommittedFiles(), COMMIT_MESSAGE).getId();
+    mockFirestoreClientMethods();
+  }
+
+  @After
+  public void after() throws InterruptedException {
+    server.shutdownNow();
+    server.awaitTermination();
+    channel.shutdownNow();
+    channel.awaitTermination(1, TimeUnit.SECONDS);
+  }
+
+  @Singleton
+  @Component(modules = {CommonModule.class, AaModule.class})
+  interface TestComponent {
+    AuthService getAuthService();
+
+    GitRepoFactory getGitRepoFactory();
+
+    InitCommand getInitCommand();
+
+    WorkspaceCommand getWorkspaceCommand();
+
+    FileUtils getFileUtils();
+
+    TextDifferencer getTextDifferencer();
+  }
+
+  private void createInitialRepo(String initialRepoFolder) {
+    fileUtils.mkdirs(initialRepoFolder);
+    GitRepo repo = gitRepoFactory.create(initialRepoFolder);
+    repo.init();
+    repo.setFakeUsersData();
+    fileUtils.writeStringUnchecked(
+        TEST_FILE_CONTENTS, fileUtils.joinPaths(initialRepoFolder, FILE_IN_HEAD));
+    fileInHeadCommitId = repo.commit(repo.getUncommittedFiles(), "Initial commit").getId();
+  }
+
+  private void initAaBase(String initialRepoFolder, String aaBaseFolder) {
+    InitCommand initCommand = component.getInitCommand();
+    InitCommand.startuposRepo.resetValueForTesting();
```

Comment:
Why do you need to reset value? It doesn't get overridden?

Reply:
Actually, this line isn't necessary. Deleted it. 
Also I deleted the same line in `CodeReviewServiceTextDiffTest`.